include/libxnvme_spec.h

```diff
@@ -1192,6 +1192,47 @@ struct xnvme_spec_cmd_common {
 };
 XNVME_STATIC_ASSERT(sizeof(struct xnvme_spec_cmd_common) == 64, "Incorrect size")
 
+/**
```

Comment:
The ``libxnvme_spec.h`` is intended for spec-definitions only, this struct should go into the Linux-backend implementation: ``xnvme_be_linux.h``.

Reply:
Okay Simon, sure. Will move this structure to xnvme_be_linux.h

Reply:
To fix the source-format caught by CI, then run: ``make source-format``.
For the build-error on Debian Buster, then you can temporarily disable that build by editing ``.github/workflows/verify.yml`` and commenting out line 280 (buster) and 281 (stretch).